app/uk/gov/hmrc/selfservicetimetopay/models/TTPSubmission.scala

```diff
@@ -23,7 +23,8 @@ case class TTPSubmission(schedule: Option[CalculatorPaymentSchedule] = None,
                          eligibilityTypeOfTax: Option[EligibilityTypeOfTax] = None,
                          eligibilityExistingTtp: Option[EligibilityExistingTTP] = None,
                          calculatorData: CalculatorInput = CalculatorInput.initial,
-                         durationMonths: Option[Int] = Some(3)) {
+                         durationMonths: Option[Int] = Some(3),
+                         eligibilityStatus: Option[EligibilityStatus] = None) {
```

Comment:
Additional parameter to our session cache stored object. Note this had the knock on effect of all of the patterns needing to be updated (as above - generally with a trailing ,_)